baseplate/clients/thrift.py

```diff
@@ -162,22 +167,54 @@ def retrying(self, **policy: Any) -> Iterator["_PooledClientProxy"]:
         )
 
 
+NANOSECONDS_PER_SECOND = 1e9
+PROM_NAMESPACE = "thriftbp_client"
+
+PROM_COMMON_LABELS = [
+    "namespace",
+    "success",
+]
+PROM_REQUESTS_LABELS = [
+    *PROM_COMMON_LABELS,
+    "exception_type",
+    "baseplate_status",
+    "baseplate_status_code",
+]
+
+PROM_LATENCY = Histogram(
+    f"{PROM_NAMESPACE}_latency_seconds",
+    "Latency of thrift client requests",
+    PROM_COMMON_LABELS,
+    buckets=default_latency_buckets,
+)
+
+PROM_REQUESTS = Counter(
+    f"{PROM_NAMESPACE}_requests_total",
+    "Total number of outgoing requests",
+    PROM_REQUESTS_LABELS,
+)
```

Comment:
It would allow some of the code below to read more smoothly if these were named similar to the way the metrics constants are named for HTTP. Maybe something like this:
`
```suggestion
REQUEST_LATENCY = Histogram(
    f"{PROM_NAMESPACE}_latency_seconds",
    "Latency of thrift client requests",
    PROM_COMMON_LABELS,
    buckets=default_latency_buckets,
)

REQUESTS_TOTAL = Counter(
    f"{PROM_NAMESPACE}_requests_total",
    "Total number of outgoing requests",
    PROM_REQUESTS_LABELS,
)
```

Reply:
I agree, fixed!